homework/style.css

```diff
@@ -1,3 +1,185 @@
 .alert-error {
```

Comment:
On a wide screen this all looks streched out. Perhaps limit the max width of the content to say, 960px or similar?

![image](https://user-images.githubusercontent.com/2788771/62378907-1b305000-b546-11e9-9390-1f29223ddbef.png)

Reply:
Not really an improvement:

![image](https://user-images.githubusercontent.com/2788771/62383417-022c9c80-b550-11e9-87d5-e5aa22ef8ee5.png)